src/offset/local/mod.rs

```diff
@@ -147,6 +150,102 @@ impl TimeZone for Local {
     }
 }
 
+#[cfg(any(unix, windows))]
+pub(crate) struct TzInfo {
+    std_offset: FixedOffset,
+    dst_offset: FixedOffset,
+    std_transition: Option<NaiveDateTime>,
```

Comment:
Is `transition` start or end? Maybe there should be a nested type here, so that we can have `std` and `dst` fields of a type `OffsetChange` (name TBD) with `offset` and `start` fields?